pymodbus/server/simulator/http_server.py

```diff
@@ -161,6 +161,7 @@ def __init__(
         self.serving: asyncio.Future = asyncio.Future()
         self.log_file = log_file
         self.site = None
+        self.runner: web.AppRunner
```

Comment:
This change should cause a mypy problem, because self.runner is now defined outside __init__

Reply:
`pylint` did complain... until I added this line with the type, which satisfies both `pylint` and `mypy`